examples/notebooks/remoteRover.ipynb

```diff
@@ -15,10 +15,12 @@
    "metadata": {},
    "source": [
     "#!rover\n",
-    "resourceScanner.CaptureFromCamera(roverBody.Camera);\n",
+    "Microsoft.DotNet.Interactive.Formatting.Formatter.ListExpansionLimit = 42;\n",
```

Comment:
I'm not clear on the purpose of this limit... Is there a need for the change here, as we still do the sectors 5x5?

Reply:
it is a one off setting to ensure autogenerate formatters expand collections up to 42 elements
we need 25 for the demo but 42 is a more meaningful limit